spec/views/content_assets/edit.html.erb_spec.rb

```diff
@@ -2,9 +2,7 @@
 
 RSpec.describe "content_assets/edit", type: :view do
   before(:each) do
-    @content_asset = assign(:content_asset, ContentAsset.create!(
-                                              title: "MyString",
-                                            ))
+    @content_asset = FactoryBot.create(:content_asset)
```

Comment:
Can we move these variables into let definitions
https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let

Reply:
Marc, I did hit trouble when I tried to do this with ContentPage tests, so let me know if you get stuck.